src/main/java/com/aorri2/goodsforyou/common/exception/ErrorCode.java

```diff
@@ -10,7 +10,10 @@ public enum ErrorCode {
 	NOT_MATCHED_EMAIL("이메일이 일치하지 않습니다."),
 
 	//Product Error Code
-	NOT_EXIST_PRODUCT("해당 상품은 존재하지 않습니다.");
+	NOT_EXIST_PRODUCT("해당 상품은 존재하지 않습니다."),
+
+	//Transaction Error Code
```

Comment:
Trade Transaction Error 라고 써 놓는게 더 명확 할 것 같긴 하네요 ㅎㅎ
취향

Reply:
```suggestion
	//Transaction Error Code
	//Trade Error Code
```
위와 같은 방식으로 변경토록 하겠습니다 😊